code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -475,10 +475,6 @@
 		else if(prob(30) && ishuman(movement_target.loc)) // mean hooman has stolen it
 			custom_emote(EMOTE_VISIBLE, "stares at [movement_target.loc]'s [movement_target] with a sad puppy-face.")
 
-/obj/item/food/snacks/meat/corgi
-	name = "Corgi meat"
-	desc = "Tastes like... well you know..."
```

Comment:
this will change the desc :gatto:

Reply:
I think "Tastes like the Head of Personnel's hopes and dreams." is way funnier